src/test/java/net/imglib2/roi/boundary/BoundaryRandomAccessTest.java

```diff
@@ -0,0 +1,91 @@
+package net.imglib2.roi.boundary;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import net.imglib2.Cursor;
+import net.imglib2.img.Img;
+import net.imglib2.img.array.ArrayImgs;
+import net.imglib2.type.logic.BitType;
+
+public class BoundaryRandomAccessTest
+{
+	@Test
+	public void testBoundaryRandomAccess8()
+	{
+		final int[] region = new int[]
+		{
+			0, 0, 1, 1, 1, 1,
+			0, 0, 1, 1, 1, 1,
+			1, 1, 1, 1, 1, 1,
+			1, 1, 1, 1, 1, 1,
```

Comment:
These are boundary pixels according to the definition given in the `Boundary` javadoc and should be detected as such. Might be useful to add a test, though...